### ISSUE TITLE
CANON_LOGREDUX env variable isn't working

### ISSUE THREAD

**Issue by frabarz** [Datawheel/canon#194]
Following the documentation, setting `export CANON_LOGREDUX=false` should turn redux console logging off. However this is not working.

Not sure if it's related, but the parsed initial state in the source code for any page generated shows 
![image](https://user-images.githubusercontent.com/11713191/46056994-c784f700-c129-11e8-90a2-9014888d3307.png)
Here, `CANON_LOGREDUX` value is the string `"false"` instead of the boolean `false`. I tried setting the value to `""` (the empty string, a "falsey" value) and didn't work either.

**Comment by davelandry** [comment]
I'm unable to reproduce this error on my machine. @frabarz it may just be your environment. Can you edit [this line](https://github.com/Datawheel/canon/blob/master/bin/server.js#L122) to say `CANON_LOGREDUX: yn(process.env.CANON_LOGREDUX),` and see if that fixes this for you?

**Comment by frabarz** [comment]
I checked this issue again, and now it's working as intended.
I suspect the issue was caused at the moment because of #123, plus the fact the HardSourceWP cache folder is now located at `node_modules/.cache`. This time I deleted it manually and worked fine.
Additionally, I now know the saved state in the SSR pages has nothing to do with how the logger is inyected; it is through webpack's DefinePlugin instead.
